fixture-monkey-datafaker/src/main/java/com/navercorp/fixturemonkey/datafaker/plugin/DataFakerPlugin.kt

```diff
@@ -0,0 +1,36 @@
+package com.navercorp.fixturemonkey.datafaker.plugin
+
+import com.navercorp.fixturemonkey.api.option.FixtureMonkeyOptionsBuilder
+import com.navercorp.fixturemonkey.api.plugin.Plugin
+import com.navercorp.fixturemonkey.api.type.Types
+import com.navercorp.fixturemonkey.datafaker.generator.DataFakerStringObjectPropertyGenerator
+import com.navercorp.fixturemonkey.datafaker.introspector.DataFakerArbitraryIntrospector
+import com.navercorp.fixturemonkey.datafaker.support.DataFakerFieldResolver
+
+class DataFakerPlugin : Plugin {
```

Comment:
I think that `DataFakerFieldResolver` can be merged within `DataFakerPlugin`.

Instead of using the map, you can implement `Matcher` as well.

Reply:
I've merged `DataFakerFieldResolver` into `DataFakerArbitraryIntrospector` and implemented the `Matcher` interface, following the `BooleanIntrospector` pattern as you suggested.

I've fixed this in commit [87bed48](https://github.com/naver/fixture-monkey/pull/1214/commits/87bed48f2cfd7f486f89ed7c95e678a4889d0ec4).